### PR TITLE
Added awesome-shopware list

### PR DESCRIPTION
**Pull request by shyim**
We started with the community to create a awesome-shopware list on github. I would like link it on general resources. ❤️

### CODE REVIEW

**Comment by mitelg** [comment]
hey @shyim 
please see here: https://github.com/shopware/devdocs/pull/677 https://github.com/shopware/devdocs/pull/430

**Comment by mitelg** [comment]
@shopwareBot we should discuss about merging this link. It is quite useful and desired by the community

File: source/developers-guide/general-resources.html [hunk 1]
```diff
@@ -31,4 +31,5 @@
     <li><a href="{{ site.url }}/developers-guide/media-optimizer/">Media Optimizer</a></li>
     <li><a href="{{ site.url }}/developers-guide/concept-cart-bundle/">Cart Concept</a></li>
     <li><a href="{{ site.url }}/developers-guide/implementing-your-own-captcha/">Implementing your own CAPTCHA</a></li>
+    <li><a href="https://github.com/shyim/awesome-shopware">A awesome-shopware list</a></li>
```
Comment (oktupol) [inline source/developers-guide/general-resources.html]: _An_ awesome... 😉

Comment (shyim) [reply source/developers-guide/general-resources.html]: Fixed ty ❤️

**Comment by mitelg** [comment]
hey @shyim 
we decided to merge your PR :+1: :tada: 
we will adjust the link a bit, to make clear, that this is not an official link and community driven :blush: :+1:

**Pull request merged**